apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -61,6 +104,39 @@ export function ShowAndTellEntryForm({
   const review = trpc.adminShowAndTell.review.useMutation();
   const isLoading = create.isLoading || update.isLoading || review.isLoading;
 
+  const [trackingStatus, setTrackingStatus] = useState({
+    active: false,
+    text: "tracking is not active",
+  });
+  useEffect(() => {
+    // Check local time is in date range
+    const now = new Date();
+    const start = new Date(giveAnHourStart);
+    const end = new Date(giveAnHourEnd);
+    const trackingStatusFrom = formatDateTime(start);
+    const trackingStatusTo = formatDateTime(end);
+
+    let active = true;
+    let verb = "is";
+    if (now < start) {
+      active = false;
+      verb = "will be";
+    } else if (now > end) {
+      active = false;
+      verb = "was";
+    }
+
+    setTrackingStatus({
+      active,
+      text: `tracking ${verb} available from ${trackingStatusFrom} to ${trackingStatusTo}`,
+    });
+  }, []);
```

Comment:
Nitpick, this could be a useMemo rather than a state + effect

Reply:
oh true. i was under the impression the form was SSR'ed but it isnt